.travis.yml

```diff
@@ -42,5 +43,5 @@ script:
   # Test fxa-auth-mailer
   - grunt templates && git status -s | (! grep 'M lib/senders/templates/')
   - grunt l10n-extract
-  # NSP check
-  - grunt nsp
+  # HACK: ignore npm audit errors for now until we get them all fixed
+  - npm audit || true
```

Comment:
if we do this we might miss a critical sec issue, we do have sentry monitoring as a back up though

Reply:
But if we don't do this we're using `nsp`, which is not being maintained or updated and guarantees that we won't see any security issues at all. How is that better than what this PR does, which is at least give us a chance of seeing issues reported by `npm audit`?

And it's a step on the road to what you're looking for, which is enabling `npm audit` properly so that CI fails when there are unresolved security reports.

Reply:
(I did look for an equivalent to the `.nsprc` mechanism for turning off just specific advisories, but didn't find anything. If you know something about that then definitely share it)

Reply:
> if we do this we might miss a critical sec issue, we do have sentry monitoring as a back up though

Are npm security alerts reported to Sentry based on latest and stage, or just prod? It'd be a bit of a shame to only see these problems when they were shipped to production.

> (I did look for an equivalent to the .nsprc mechanism for turning off just specific advisories, but didn't find anything. If you know something about that then definitely share it)

@philbooth - [ignoring particular packages is a requested feature](https://github.com/npm/npm/issues/20565).

I too would feel much better if we didn't have the `|| true` and fixed the mocha problem, but can also understand ditching nsp now and cleaning up in a follow on. If npm security alerts can occur based on latest, that should provide sufficient protection until we get mocha updated.

Reply:
Do you have github vulnerability alerts enabled too?

Reply:
> Do we have github vulnerability alerts (should an email per vuln or a digest) enabled for FxA too?

We have this, is that what you mean?

<img width="680" alt="Screen shot of GitHub alert settings for fxa-auth-server" src="https://user-images.githubusercontent.com/64367/43163529-516d78dc-8f86-11e8-9c0c-52edccc4ddaf.png" />

There's no email though AFAIK, it just shows you the alerts via the web UI.

Reply:
> There's no email though AFAIK

The emails go to the github-owners mailing list; I wonder if there's some way to customize them for own own repos so they go to fxa-staff or similar...

Reply:
Ah, okay. I just experimented with adding `fxa-core` to the alerts for the auth server explicitly, we can see if it makes a difference...

Reply:
(won't go to `fxa-staff@` I realise, but maybe it will circulate them more widely)

Reply:
> I wonder if there's some way to customize them for own own repos

Haha oh dear, I must still be waking up - you posted a screenshot of the form that lets you customize this and I'm all like "I wonder if it's possible to customize this...".  Thanks @philbooth!